lisa/tools/gcc.py

```diff
@@ -50,5 +50,26 @@ def get_version(self) -> VersionInfo:
 
     def _install(self) -> bool:
         posix_os: Posix = cast(Posix, self.node.os)
-        posix_os.install_packages("gcc")
+        if isinstance(posix_os, BSD):
+            posix_os.install_packages("lang/gcc")
+        else:
+            posix_os.install_packages("gcc")
         return self._check_exists()
+
+    def install_cpp_compiler(self) -> None:
+        node_os = self.node.os
+        if isinstance(node_os, Debian):
+            node_os.install_packages("g++")
+        elif (
+            isinstance(node_os, Fedora)
+            or isinstance(node_os, Suse)
+            or isinstance(node_os, CBLMariner)
+        ):
+            node_os.install_packages("gcc-c++")
+        elif isinstance(node_os, BSD):
+            # BSD installs g++ with lang/gcc
+            return
```

Comment:
Leave a log line, so we know what happened. If log works, it doesn't need code comment.

Reply:
removing, see other comment